client/src/views/account/AccountSettingsPage.vue

```diff
@@ -0,0 +1,208 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
```

Comment:
Why not used the settings page and add tabs if Parsec Account is activated and we're logged in?
Also why is the other one a dialog and this one is not?